### DISCUSSION TITLE
Public Key

### DISCUSSION THREAD

**Discussion by CarolineChiefari** [CarolineChief/Tafe240261#6] [General]
Hey,

Don't forget to get me your public key so I can it to the repo.

Thanks

**Comment by Hayden-Badenhop** [comment]
Hi Caroline, 

Sorry for the late response, my public key is 

AAAAB3NzaC1yc2EAAAADAQABAAACAQDL5Wi8nh6Oh7t2XOTpwlJLSq7Mr+VnopzjbzLriJAlofvzb7RQUO0dz8wmIjuAoc+DzDo4r4nTjE8tyf2pTu2TMDrF4Yu9HMAWQw6eUjAwtOG/jsqzM403pmwsdLWXqWFsfq5Hdh2/Ze3luwBrnGnhIod26xLgIBQuiGF/hvaEoFlmQI60WZmHHYOUZ/TwShZ6eoP80zbVOnwb457IFNG6mvPGcy9MM1W/N6H8P+UH1Wt6KdxDELJf9//euDe0IZ9p3OY7a3xV5Xo8wKQSGlDShv+pgkvSza8nMa5AufAVdsnUO5yf12QvHhbsclCf3D8Xv0DvnGIDgwZRQhL7Hq2rWaDzGQqfucQso0LCyDtYMrA2r2PdDVycxeuCqtTSuFsXXfMJvnh9yKrml4Tt5L3zwdFadwx8QDwal2dcBs/uoPwkCzSJ+kUZ+OA89Qrx6a6IxvHrG5bsltBxUjgnbwpLzLBZ4oxfsbUORrm/m5Tq5MZTdMh1dDkoX0jEwx9F0dWVAnfNsT4juJSqpdT28bD1HNfPsff9VhG5oXGonooUP8DBugQ1TQ2L3vFv8Fj1hKSKPsRZp6RLy4omx1Znh1uAt6sGpi82CdVS6WwL4SG9wzq9fY8z7OXNzoEiDb5zFLe3e44rJg/mpePxGiWTKZtBoEbTd9gj08sN1AMms+6Xaw== Hayden.BADENHOP@student.tafesa.edu.au

**Reply by CarolineChiefari** [reply]
Thanks Hayden your public key has been added to the repo.
Caroline
